### ISSUE TITLE
shortcuts without modifier keys prevent writing that key in inputs

### ISSUE THREAD

**Issue by nfcampos** [gaearon/redux-devtools-dock-monitor#29]
``` js
<DockMonitor toggleVisibilityKey='s'>
  <LogMonitor/>
</DockMonitor>
```

Hi,

setting a shortcut like that (without a modifier key) prevents me from typing that letter in inputs (because  `handleKeyDown()` calls `event.preventDefault()`)

one solution for this would be to add this line
`if (e.target !== document.body) return`
to `handleKeyDown()` because if an input is focused the target of the keydown event is that input, if no input is focused the target of the event is the body

another solution would be to `return` if the tag name of the target of the event is input, textarea, etc.

would you be open to doing one of these things, or some other way of not triggering the shortcuts while an input is focused?

Thanks,
Nuno

(also, thanks a lot for all you've done with redux,etc it plays a big part in how i write apps now :smiley: )

**Comment by gaearon** [comment]
Thanks for kind words.
Why not just set a shortcut with modifier key?

**Comment by nfcampos** [comment]
that was just because this app i'm working on has other shortcuts like that (without modifier key) and it was just to be consistent. (the shortcut library https://craig.is/killing/mice i use for the other shortcuts does this thing of not triggering them when focused on an input)

**Comment by gaearon** [comment]
I'm happy to add a check [like this](https://github.com/ccampbell/mousetrap/blob/ab4327c82b1b2e38b1d8ac5576f32a49e4441ea3/mousetrap.js#L975). Want to send a PR?

**Comment by nfcampos** [comment]
done :)

**Comment by gaearon** [comment]
Fixed by #30.
